src/main/java/lotto/domain/Lotto.java

```diff
@@ -0,0 +1,52 @@
+package lotto.domain;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+public class Lotto {
+    public static final int PRICE = 1000;
+    private Set<LottoNumber> lotto;
+    private int winningNumberCount;
+
+    public Lotto(List<LottoNumber> lotto) {
+        this.lotto = new HashSet<LottoNumber>(lotto);
+
+        isValidSize();
+    }
+
+    private void isValidLotto(List<String> lotto) {
+        Set<Integer> set = new HashSet<>();
+
+    }
+
+    private void isValidSize(){
+        if(this.lotto.size() != 6) throw new IllegalArgumentException("Lotto must be contain unique 6 numbers");
+    }
+
+    @Override
+    public String toString() {
+        String lottoStr = lotto.stream()
+                .map(i -> i.getNumberString())
+                .collect(Collectors.joining(", "));
+
+        return String.format("[%s]", lottoStr);
```

Comment:
List의 toString을 활용하셔도 원하시는 포맷으로 출력됩니다:)
```suggestion
        return lotto.stream()
                .map(LottoNumber::getNumberString)
                .collect(Collectors.toList())
                .toString();
```